cmd/git-sync/main.go

```diff
@@ -123,7 +123,7 @@ func envFloat(key string, def float64) float64 {
 
 func main() {
 	setFlagDefaults()
-
+    flag.Set("v","2")
```

Comment:
Is this not configurable via a command line arg?

Reply:
it can be done. but it has to be defaulted to 2 either here or from the caller to always print out the detailed logs.

Reply:
Shouldn't it be less verbose by default on the principle of configuring the bare minimum to operate/administer correctly?

Reply:
The objective of increasing the log level here to print detailed logs in order to figure out why there was no activity in the git-sync for an extended period of time. Once after identifying the root-cause, the verbose level will be defaulted.

Reply:
I will let @asobti take the final call.

Reply:
I think you might have misunderstood my first question. Can we not set the log level via a command line argument when starting git-sync? We should not have to make a code change each time we want to change the log level.

Can we not do something like:
```
/git-sync --git-dir=/foo/bar/ -v=8
```